DistributedLeaseManager.AzureCosmosDb/DistributedLeaseCosmosDbOptions.cs

```diff
@@ -4,4 +4,5 @@ public class DistributedLeaseCosmosDbOptions
 {
     public string DatabaseName { get; set; } = string.Empty;
     public string ContainerName { get; set; } = string.Empty;
+    public string PartitionKeyPath { get; set; } = string.Empty;
```

Comment:
As you've noted, it would be great to have a default value of `/pk` here to avoid introducing a breaking change.

Note that this may result in a need to create new extension methods in `ServiceCollectionExtensions` instead of modifying the existing ones, since there could be ambiguities between calls to the methods with fewer arguments and the methods with an optional argument. My suggestion for the new methods is to accept a configuration delegate:

```csharp
public static IServiceCollection AddCosmosDbDistributedLeaseManager(
    this IServiceCollection services,
    string cosmosDbConnectionString,
    Action<DistributedLeaseCosmosDbOptions> optionsConfiguration)
{
    // ...

    return services.AddCosmosDbDistributedLeaseManager(optionsConfiguration);
}

public static IServiceCollection AddCosmosDbDistributedLeaseManager(
    this IServiceCollection services,
    Action<DistributedLeaseCosmosDbOptions> optionsConfiguration)
{
    // ...

    services.Configure<DistributedLeaseCosmosDbOptions>(optionsConfiguration);

    // ...
}
```